workflows/illumina_PE/monkeypox_illumina_PE.wdl

```diff
@@ -210,6 +218,42 @@ task call_variants {
   }
 }
 
+task pipeline_header_vcf {
+  input {
+    String accession
+    String container_registry
+    File initial_vcf
+  }
+
+  Int disk_size = ceil(size(initial_vcf, "GB") + 20)
+
+  command <<<
+    pipeline_release=$(curl -s GET https://api.github.com/repos/DNAstack/monkeypox-processing-pipeline/tags | jq -r '.[].name' | head -1)
+
+    # Write out the header to be modified
+    bcftools view -h ~{initial_vcf} > header
+
+    # Edit the header 
+    line="$(grep -n '#CHROM' header | cut -f1 -d:)"
+    sed -i "$(($line))i\##pipeline=https://github.com/DNAstack/monkeypox-processing-pipeline/releases/tag/$pipeline_release" header
```

Comment:
Clever way to get around the fact that you can't use `{}`!

Since `miniwdl check` complains about this, an alternative is:
`sed -i "$((line))i\##pipeline=https://github.com/DNAstack/monkeypox-processing-pipeline/releases/tag/$pipeline_release" header`

or

`sed -i $line"i\##pipeline=https://github.com/DNAstack/monkeypox-processing-pipeline/releases/tag/$pipeline_release" header`

`sed` will let you break up the command with runs of quoted/unquoted chunks, as long as the full substitution chunk resolves to something without spaces.